script.js

```diff
@@ -182,7 +182,7 @@ document.addEventListener('keydown', function(e) {
         const lightboxModal = document.querySelector('#lightboxModal');
         if (!globalPopup || globalPopup.style.display === 'none') {
             if (document.activeElement === lightboxModal) return;
-            if (interruptButton.style.display === 'block') {
+            if (interruptButton && interruptButton.style.display === 'block') {
```

Comment:
```suggestion
            if (interruptButton?.style.display === 'block') {
```
This can be simplified.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining